src/test/java/com/eleks/academy/whoami/core/impl/PersistentPlayerTest.java

```diff
@@ -0,0 +1,22 @@
+package com.eleks.academy.whoami.core.impl;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.Future;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.concurrent.Future'